run.js

```diff
@@ -29,9 +29,9 @@ async function main() {
       .map(item => relative('packages', item).split('/')[0]);
 
     matches = Array.from(new Set(changed));
+    matches.push('now-cli'); // always test Now CLI
```

Comment:
Won't this run now-cli tests _twice_ if now-cli code has actually changed? I think it should be `changed.add('now-cli')` instead.

Reply:
@TooTallNate Yep! I fixed it it 😄